packages/stack/src/views/Stack/Card.tsx

```diff
@@ -574,7 +574,13 @@ export default class Card extends React.Component<Props> {
                 this.offset,
                 cond(
                   this.distance,
-                  divide(this.gesture, this.distance),
+                  divide(
+                    multiply(
+                      this.gestureUntraversed,
+                      I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
```

Comment:
Instead of `TRUE_NODE`, can we do `PLUS_ONE_NODE`?

Reply:
yes we can but since I made this decision in many places I want to do it in follow up